modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaExtensionModelParserTestCase.java

```diff
@@ -234,4 +294,49 @@ private static class ExtensionAnnotatedWithHighMMV {
     String extensionParameter;
   }
 
+  private static class SimplePojoWithTime {
+
+    @Parameter
+    private String name;
+
+    @Parameter
+    private Integer age;
+
+    @Parameter
+    private LocalDateTime dateTime;
+  }
+
+  protected static ExtensionModel loadExtension(Class<?> clazz, ExtensionModelLoader loader, ArtifactCoordinates coordinates) {
```

Comment:
I borrowed this code from ParameterizedExtensionModelTestCase

Reply:
why not `MuleExtensionUtils#loadExtension`? that one is already being used in tests in this module...